efcsydney/efcsydney-roster
# #341 Introduce TestCafe

#### Description

Ability to use TestCafe as our E2E testing tool

#### QA Steps

* Checkout to this `#341` branch on your local machine.
* Install the new required NPM packages with `yarn install`.
* Execute `yarn test:functional` to see if it can execute test **on QA env** successfully.
* Execute `NODE_ENV=development yarn test:functional` to see if it can execute test **on local env** 
successfully. (Need to `yarn start` to have the local server first)
(use `set NODE_ENV=development&&yarn test:functional` on windows)

#### Acceptance Criteria

* Ensure the testing process is working smoothly as you expected
* Ensure the test case itself is easy to understand.
